Jellyfin.Server/Program.cs

```diff
@@ -210,7 +152,7 @@ private static async Task StartServer(IServerApplicationPaths appPaths, StartupO
 
                 try
                 {
-                    await host.StartAsync(_tokenSource.Token).ConfigureAwait(false);
+                    await host.StartAsync().ConfigureAwait(false);
```

Comment:
Why remove the token?

Reply:
The token was only used in our listeners for SIGTERM/Ctrl+C, which is handled by the ConsoleLifetime